include/solvers/simulation_parameters.h

```diff
@@ -190,15 +190,18 @@ class SimulationParameters
         not(multiphysics.vof_parameters.conservation.monitoring))
       {
         throw std::logic_error(
-          "Inconsistency in .prm!\n in subsection VOF, with sharpening type = adaptative\n "
+          "Inconsistency in .prm!\n in subsection vof, with sharpening type = adaptative\n "
```

Comment:
the subsection name in the prm is  capitalized I think
```suggestion
          "Inconsistency in .prm!\n in subsection VOF, with sharpening type = adaptative\n "
```